ms/make_lib.bat

```diff
@@ -6,41 +6,24 @@ set ACV_LIB_PATHS=
 rem Visual Studio wants absolute paths in some cases
 set ACV_ROOT_PATH_REL=%~dp0..\
 for %%i in ("%ACV_ROOT_PATH_REL%") do SET "ACV_ROOT_PATH=%%~fi
-if "%STATIC_BUILD%" == "TRUE" (
-  set PROJ_CONFIG=static
-) else (
-  set PROJ_CONFIG=shared
-)
 
-if "%OFFLINE_BUILD%" == "TRUE" (
-  set PROJ_CONFIG=%PROJ_CONFIG%_offline
-) else (
-  if [%LIBCURL_DIR%] == [] (
-    echo "curl dir not specified - attempting to use murl and link to ssl..."
-	  if [%SSL_DIR%] == [] (
-	    echo "No SSL dir specified. Curl directory, or SSL dir if using Murl, must be specified. exiting..."
-      goto :error
-	  ) else (
-      set ACV_LIB_PATHS=%SSL_DIR%\lib
-	    set ACV_INC_PATHS=%SSL_DIR%\include
-	  )
-  ) else (
-    set ACV_LIB_PATHS=%LIBCURL_DIR%\lib
-   	set ACV_INC_PATHS=%LIBCURL_DIR%\include
-  )
-)
+set PROJ_CONFIG=shared
 
-if [%SAFEC_DIR%] == [] (
-  set PROJ_CONFIG=%PROJ_CONFIG%_no_safec
-  set ACV_INC_PATHS=%ACV_INC_PATHS%;%ACV_ROOT_PATH%\safe_c_stub\include
+if [%LIBCURL_DIR%] == [] (
+  set PROJ_CONFIG=offline
 ) else (
-  set ACV_LIB_PATHS=%ACV_LIB_PATHS%;%SAFEC_DIR%
-  set ACV_INC_PATHS=%ACV_INC_PATHS%;%SAFEC_DIR%\include
+  set ACV_LIB_PATHS=%LIBCURL_DIR%\lib
+  set ACV_INC_PATHS=%LIBCURL_DIR%\include
 )
 
 if [%LIBCURL_DIR%] == [] (
-  set PROJ_CONFIG=%PROJ_CONFIG%_murl
-  set ACV_INC_PATHS=%ACV_INC_PATHS%;%ACV_ROOT_PATH%\murl
+  if [%SSL_DIR%] == [] (
+    echo "No SSL dir specified. Must be provided for online builds. exiting..."

```

Comment:
Remove ^M (carriage return)

Reply:
I am a bit vague on how Git displays line endings, but .bat files need CRLF line endings for labels to work properly; unless that is not the issue here?

Reply:
I think they show up depending on the editor used (typically a none plan text format).
Not a big deal.